### ISSUE TITLE
White name bug

### ISSUE THREAD

**Issue by snsty** [ReflxctionDev/NameFormatter#2]
Makes your name white. Here is an example from Musical_Cookie (because I'm too lazy to take a screenshot myself)! https://media.discordapp.net/attachments/363721897743089671/424357281140834304/2018-03-17_00.03.46.png?width=1296&height=676

**Comment by snsty** [comment]
@ReflxctionDev

**Comment by Revxrsal** [comment]
Yeah, I'm aware of that issue. Apparently it's because the client renders the display name, then hypixel modifies the display name again for ranks. The only solution is to make it completely client side, and I'm working on that. Thanks
